infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -133,6 +178,14 @@ type RegistryFilePersistence struct {
 	S3AdditionalKwargs *map[string]string `json:"s3_additional_kwargs,omitempty"`
 }
 
+// RegistryDBStorePersistence configures the DB store persistence for the registry service
+type RegistryDBStorePersistence struct {
+	// +kubebuilder:validation:Enum=sql;snowflake.registry
+	Type          string                       `json:"type,omitempty"`
+	SecretRef     *corev1.LocalObjectReference `json:"secretRef,omitempty"`
+	SecretKeyName string                       `json:"secretKeyName,omitempty"`
+}
+
 // PvcConfig defines the settings for a persistent file store based on PVCs.
```

Comment:
did we miss the `ValidRegistryDBStorePersistenceTypes` list?

Reply:
Done